### PR TITLE
data privacy: update ref app link and personal data annotations

### PR DESCRIPTION
**Pull request by mnkiefer**
- Fix link:
   The new *Incidents Management reference sample app* to be used as of now is:
   https://github.com/cap-js/incidents-app

### CODE REVIEW

**Review by RamIndia — APPROVED** [review]
Looks Good :)

File: guides/data-privacy/annotations.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Following the [best practice of separation of concerns](../domain-modeling#separ
 ::: code-group
 
 ```cds [srv/data-privacy.cds]
-using { sap.capire.incidents as db } from '@capire/incidents';
+using { sap.capire.incidents as db } from './processor-service';
```
Comment (sjvans) [inline guides/data-privacy/annotations.md:37]:
```suggestion
using { sap.capire.incidents as db } from '../db/schema';
```
this guide is written as if you'd clone incidents-app and work in that repo directly

Comment (sjvans) [reply guides/data-privacy/annotations.md:37]: but there are no addresses in incidents-app... we'll need to think of a solution

Comment (sjvans) [reply guides/data-privacy/annotations.md:37]: @mnkiefer let's discuss this in the next sync

File: guides/data-privacy/annotations.md [hunk 2]
```diff
@@ -21,22 +21,49 @@ In order to automate audit logging, personal data management, and data retention
 
 ## Reference App Sample { #annotated-model }
 
-In the remainder of this guide, we use the [Incidents Management reference sample app](https://github.com/SAP-samples/cap-sample-incidents-mgmt) as the base to add data privacy and audit logging to.
+In the remainder of this guide, we use the [Incidents Management reference sample app](https://github.com/cap-js/incidents-app) as the base to add data privacy and audit logging to.
```
Comment (sjvans) [inline guides/data-privacy/annotations.md:24]:
```suggestion
In the remainder of this guide, we use the [Incidents Management reference sample app](https://github.com/cap-js/incidents-app) with some data model extensions (see below) as the base to add data privacy and audit logging to.
```

**Comment by sjvans** [comment]
data model extensions as expandable details:

Screenshot 2023-10-19 at 17 06 56

Screenshot 2023-10-19 at 17 04 21

**Review by danjoa — APPROVED** [review]
👍 Thanks Mara!

File: guides/data-privacy/annotations.md [hunk 3]
```diff
@@ -21,22 +21,49 @@ In order to automate audit logging, personal data management, and data retention
...
 
-<img src="./assets/Incidents-App.drawio.svg" style="zoom:111%;" />
+In order to showcase sensitive data and data subject details, we need to extend the data model a bit as follows:
+
+```cds [db/extensions.cds]
+using from '@capire/incidents';
+using {
+  cuid,
+  managed
+} from '@sap/cds/common';
```
Comment (sjvans) [inline guides/data-privacy/annotations.md:37]:
```suggestion
using { cuid, managed } from '@sap/cds/common';
```

File: guides/data-privacy/annotations.md [hunk 4]
```diff
@@ -21,22 +21,49 @@ In order to automate audit logging, personal data management, and data retention
...
+In essence, in all our entities we search for elements which carry personal data, such as person names, birth dates, etc., and tag them accordingly.
+All found entities are classified as either *Data Subjects*, *Subject Details* or *Related Data Objects*.
 
 Following the [best practice of separation of concerns](../domain-modeling#separation-of-concerns), we annotate our domain model in a separate file *srv/data-privacy.cds*, which we add to our project and fill it with the following content:
 
 ::: code-group
 
 ```cds [srv/data-privacy.cds]
-using { sap.capire.incidents as db } from '@capire/incidents';
+using {sap.capire.incidents as my} from '../db/extensions';
```
Comment (sjvans) [inline guides/data-privacy/annotations.md:64]:
```suggestion
using { sap.capire.incidents as my } from '../db/extensions';
```

**Pull request merged**